deploy/charts/cert-manager/templates/webhook-mutating-webhook.yaml

```
@@ -33,21 +27,15 @@ webhooks:
           - UPDATE
         resources:
           - "*/*"
-    {{- if $isV1AdmissionRegistration }}
     admissionReviewVersions: ["v1", "v1beta1"]
```

```suggestion
    admissionReviewVersions: ["v1"]
```

Should we remove the v1beta1 here too?

---

No that breaks Kubernetes 1.16